install.sh

```diff
@@ -75,7 +75,9 @@ fi
 echo -e "${PACYELLOW}┌────────────────────────┐\n│   ${PACCYAN}Pacstall Installer${PACYELLOW}   │\n└────────────────────────┘${NC}"
 
 echo
-if [[ -z "$(find -H /var/lib/apt/lists -maxdepth 0 -mtime -7)" ]]; then
+eval "$(apt-config shell Cache Dir::Cache)"
+eval "$(apt-config shell List Dir::State::Lists)"
+if [[ -z "$(find -H "${Cache}/${List}" -maxdepth 0 -mtime -7)" ]]; then
```

Comment:
I dont like the evals
```suggestion
Cache="$(apt-config shell Cache Dir::Cache | cut -d"'" -f2)"
List="$(apt-config shell List Dir::State::Lists | cut -d"'" -f2)"
if [[ -z "$(find -H "/${Cache}/${List}" -maxdepth 0 -mtime -7)" ]]; then
```